controllers/nstemplateset/nstemplatetier_test.go

```diff
@@ -86,11 +90,51 @@ parameters:
 	assert.Equal(t, "johnsmith", obj[0].GetName())
 }
 
+func TestProcessWithTTR(t *testing.T) {
+	// given
+	s := runtime.NewScheme()
+	err := apis.AddToScheme(s)
+	require.NoError(t, err)
+	ttRev := createTierTemplateRevision("test-ttr")
+	crq := newTestCRQ("600")
+	objectString, err := json.Marshal(crq.Object)
+	require.NoError(t, err)
+	ttRev.Spec.TemplateObjects = append(ttRev.Spec.TemplateObjects, runtime.RawExtension{Raw: []byte(objectString)})
+	ttRev.Spec.Parameters = []toolchainv1alpha1.Parameter{
+		{Name: "DEPLOYMENT_QUOTA",
+			Value: "600"},
+	}
+
+	tierTemplate := &tierTemplate{
+		ttr: ttRev,
+	}
+	ttrObj, err := tierTemplate.process(s, map[string]string{
+		SpaceName: "johnsmith",
+	})
+
+	// then
+	require.NoError(t, err)
+	require.Len(t, ttrObj, 1)
+	require.Equal(t, "for-johnsmith-deployments", ttrObj[0].GetName())
+	require.Equal(t, &expectedCRQ, ttrObj[0])
+	t.Run("test convert parameters to map", func(t *testing.T) {
+		paramM := tierTemplate.convertParametersToMap(map[string]string{
+			SpaceName: "johnsmith",
+		})
+		expectedParameters := map[string]string{
+			"DEPLOYMENT_QUOTA": "600",
+			"SPACE_NAME":       "johnsmith",
+		}
+		require.Equal(t, expectedParameters, paramM)
```

Comment:
minor - but this can be checked in the parent test as well right ? We should check that the `tiertemplate.process` evaluates the parameters correctly as well. It can even be done just in the parent test IMO without the need to test specifically `convertParametersToMap`. Or you can create a specific unit test for `convertParametersToMap` that covers all the scenarios of this function.

Reply:
its a extra, i have removed it, this  is already being tested at line [565](https://github.com/codeready-toolchain/member-operator/pull/680/files#diff-e802d4e76fc52b3d9a849d796bd916ae2fcdeea670a440aef10a32ff61963931R565),
Also in this function, i am checking the expected object  crq, where these paramters are being checked.